ingest/src/main/java/com/microsoft/azure/kusto/ingest/utils/ContainerWithSas.java

```diff
@@ -0,0 +1,32 @@
+package com.microsoft.azure.kusto.ingest.utils;
+
+import com.azure.core.http.HttpClient;
+import com.azure.storage.blob.BlobContainerClient;
+import com.azure.storage.blob.BlobContainerClientBuilder;
+
+public class ContainerWithSas {
+    private final String sas;
+    private final BlobContainerClient container;
+
+    public ContainerWithSas(String url, HttpClient httpClient) {
+        String[] parts = url.split("\\?");
+        this.sas = '?' + parts[1];
```

Comment:
Just making sure that there isn't any better way to already parse it. If not, then first make sure there are exactly 2 parts, and give them meaningful names

Reply:
will name the parts